lighthouse-core/lib/i18n/i18n.js

```diff
@@ -108,6 +110,8 @@ const UIStrings = {
   otherResourceType: 'Other',
   /** Label for a row in a data table; entries will be the total number and byte size of all third-party resources loaded by a web page. 'Third-party resources are items loaded from URLs that aren't controlled by the owner of the web page. */
   thirdPartyResourceType: 'Third-party',
+  /** Label used to identify a value in a table where many individual values are aggregated to a single value, for brevity. "Other resources" could also be read as "the rest of the resources". Resource refers to network resources requested by the browser. */
+  otherValue: 'Other resources',
```

Comment:
Now that it's here in a generic place, WDYT about `subItemOther` or something more descriptive for how it's used?

Reply:
why specify subItem? I don't think the text would change for a row.

Reply:
OK, `rowOther` then?

I don't think we should ever create a non-subitem row that says this though, so now that we're on the subject, I actually like `subItemOther` even more :)

Reply:
I'm going with `otherResourcesLabel` since the text is specific to "resources"